migration/1680507337701-add_some_donations_to_db.ts

```diff
@@ -0,0 +1,364 @@
+    toWalletAddress: '0xc1460588cA2BcAEB28c80327413e91655505A784',
+    projectId: 2351,
+    transactionId:
+      '0x7d62181f89d5abde19d4159a59a5e2bfee17860207b0e5fb026e93d49cc98fdd',
+    currency: 'USDC',
+    amount: 793.2,
+    valueUsd: 793.2,
+    transactionNetworkId: NETWORK_IDS.MAIN_NET,
+    createdAt: new Date(1679915447000),
+  },
+
+  // The matic one has 6 donations:  https://polygonscan.com/tx/0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f
+
+  {
+    fromWalletAddress,
+    toWalletAddress: '0x236daa98f115caa9991a3894ae387cdc13eaad1b',
+    projectId: 251,
+    transactionId:
+      '0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f',
+    currency: 'MATIC',
+    amount: 5.0667,
+    valueUsd: 5.32,
+    transactionNetworkId: NETWORK_IDS.POLYGON,
+    createdAt: new Date(1679915779000),
+  },
+  {
+    fromWalletAddress,
+    toWalletAddress: '0x924393ea45da424643c89f99bfdfd707b1fed9ae',
+    projectId: 2341,
+    transactionId:
+      '0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f',
+    currency: 'MATIC',
+    amount: 5.0667,
+    valueUsd: 5.32,
+    transactionNetworkId: NETWORK_IDS.POLYGON,
+    createdAt: new Date(1679915779000),
+  },
+  {
+    fromWalletAddress,
+    toWalletAddress: '0xc1460588cA2BcAEB28c80327413e91655505A784',
+    projectId: 2351,
+    transactionId:
+      '0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f',
+    currency: 'MATIC',
+    amount: 5.0667,
+    valueUsd: 5.32,
+    transactionNetworkId: NETWORK_IDS.POLYGON,
+    createdAt: new Date(1679915779000),
+  },
+
+  {
+    fromWalletAddress,
+    toWalletAddress: '0x236daa98f115caa9991a3894ae387cdc13eaad1b',
+    projectId: 251,
+    transactionId:
+      '0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f',
+    currency: 'WETH',
+    amount: 0.001667,
+    valueUsd: 3.01,
+    transactionNetworkId: NETWORK_IDS.POLYGON,
+    createdAt: new Date(1679915779000),
+  },
+  {
+    fromWalletAddress,
+    toWalletAddress: '0x924393ea45da424643c89f99bfdfd707b1fed9ae',
+    projectId: 2341,
+    transactionId:
+      '0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f',
+    currency: 'WETH',
+    amount: 0.001667,
+    valueUsd: 3.01,
+    transactionNetworkId: NETWORK_IDS.POLYGON,
+    createdAt: new Date(1679915779000),
+  },
+  {
+    fromWalletAddress,
+    toWalletAddress: '0xc1460588cA2BcAEB28c80327413e91655505A784',
+    projectId: 2351,
+    transactionId:
+      '0x7e34422db03a384def1d7eb69d95e063b84fb025a39d6cce42cefd54a903e35f',
+    currency: 'WETH',
+    amount: 0.001667,
+    valueUsd: 3.01,
+    transactionNetworkId: NETWORK_IDS.POLYGON,
+    createdAt: new Date(1679915779000),
+  },
+];
+
+export class addSomeDonationsToDb1680507337701 implements MigrationInterface {
+  async up(queryRunner: QueryRunner): Promise<void> {
+    const environment = config.get('ENVIRONMENT') as string;
+
+    if (environment !== 'local') {
```

Comment:
this wont let the donation be created in production.
Staging and production has Environment as "staging" and "production" respectively.

Reply:
Thanks for mentioning, I fixed it